src/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,127 +1,43 @@
 import React, { useEffect, useState } from 'react';
 import axiosWithAuth from '../../../utils/axiosWithAuth';
-import { Statistic, Row, Col, Table } from 'antd';
-// TODO: update page styling and functionality, see wireframes/NewDesignProposition/AdminWireframes/Dashboard
 
-const columns = [
-  {
-    title: 'Ticket ID',
-    dataIndex: 'ticketID',
-    filters: [
-      {
-        text: 'Example',
-        value: 'Example',
-      },
-      {
-        text: 'Jim',
-        value: 'Jim',
-      },
-      {
-        text: 'Submenu',
-        value: 'Submenu',
-        children: [
-          {
-            text: 'Green',
-            value: 'Green',
-          },
-          {
-            text: 'Black',
-            value: 'Black',
-          },
-        ],
-      },
-    ],
-    // specify the condition of filtering result
-    // here is that finding the name started with `value`
-    onFilter: (value, record) => record.ticketID.indexOf(value) === 0,
-    sorter: (a, b) => a.ticketID - b.ticketID,
-    sortDirections: ['descend'],
-  },
-  {
-    title: 'Message',
-    dataIndex: 'message',
-    filters: [
-      {
-        text: 'London',
-        value: 'London',
-      },
-      {
-        text: 'New York',
-        value: 'New York',
-      },
-    ],
+/**
+ * Khaleel Musleh
+ * Created Dashboard Compoenent and set it to display the user role, this component is for future use in displaying dashboard based on user role, each dashboard will be set differently
+ * based on user role.
+ */
 
-    onFilter: (value, record) => record.ticketType.indexOf(value) === 0,
-  },
-  {
-    title: 'Date Submitted',
-    dataIndex: 'dateSubmitted',
-    defaultSortOrder: 'descend',
-    sorter: (a, b) => a.dateSubmitted < b.dateSubmitted,
-  },
-];
+const Dashboard = () => {
+  const [role, setRole] = useState([]);
 
-function onChange(pagination, filters, sorter, extra) {
-  // console.log('params', pagination, filters, sorter, extra);
-}
-
-// TODO: make Ant Design Statistics pull ticket totals from ticket tables
-const Dashboard = props => {
-  const [tickets, setTickets] = useState([]);
+  const getAccounts = () => {
+    axiosWithAuth()
+      .get('/profile/current_user_profile/')
+      .then(res => {
+        setRole(
+          res.data.role_id === 1
+            ? 'Super Admin'
+            : res.data.role_id === 2
+            ? 'Admin'
+            : res.data.role_id === 3
+            ? 'Mentor'
+            : res.data.role_id === 4
+            ? 'Mentee'
+            : res.data.role_id === 5
+            ? 'Developer'
+            : 'pending'
+        );
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
currently this error is simply being logged out to the console. lets return handle it for the user properly. 

You could `console.error` here for intents and purposes of shipping, however, I'd recommend showing the user that something went wrong. eventually that's what we want to do with our errors. why not now?

Reply:
no need to fix right away, but I'd love to know more about what you plan to do in the future to handle errors back to users.

Reply:
Ive changed it to console.error, its much better like this, thank you for noticing it

Reply:
Im now using state slice and dispatch to handle errors for API requests